src/SprykerEco/Zed/UnzerGui/Communication/Form/Constraint/UnzerCredentialsConstraintValidator.php

```diff
@@ -32,17 +33,18 @@ public function validate($value, Constraint $constraint): void
             ));
         }
 
-        $unzerValidationResponseTransfer = $constraint->getUnzerFacade()->validateUnzerCredentials($value);
+        $unzerCredentialsResponseTransfer = (new UnzerCredentialsResponseTransfer())->setUnzerCredentials($value)->setIsSuccessful(true);
+        $unzerCredentialsResponseTransfer = $constraint->getUnzerFacade()->validateUnzerCredentials($unzerCredentialsResponseTransfer);
```

Comment:
this approach is wrong
we need to use `*RequestTransfer` as incoming parameter to facade